### PR TITLE
Draft: Migrate from pino 6 to pino 7

### PR DESCRIPTION
**Pull request by Zn250g**
Noticed issues:
  - some spnapshot tests are not stable. In one out of 10 cases, some snapshot tests (e.g., 'destroy.js') change the order of output lines

### CODE REVIEW

File: create-reporter/index.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
 import pino from 'pino'
 import os from 'os'
+import path from "path"
+import tty from 'tty'
 
-import { humanFormatter } from '../human-formatter/index.js'
+export const PATH_TO_PRETTIFYING_PINO_TRANSPORT = '../human-formatter/index.js';
```
Comment (ai) [inline create-reporter/index.js:6]:
```suggestion
export const PATH_TO_PRETTIFYING_PINO_TRANSPORT = '../human-formatter/index.js'
```

Comment (ai) [reply create-reporter/index.js:6]: Do we need to use `path.join(dir)`? Relative paths are bad to export.

File: create-reporter/index.js [hunk 2]
```diff
@@ -197,26 +199,50 @@ const REPORTERS = {
   }
 }
 
+// https://github.com/alexeyraspopov/picocolors/blob/147fbdbd5aea8e49d246fd5faee5e1c4f97d560a/picocolors.js#L3
```
Comment (ai) [inline create-reporter/index.js:202]: Why we can’t use `picocolors`?

File: create-reporter/index.js [hunk 3]
```diff
@@ -1,7 +1,14 @@
 import pino from 'pino'
 import os from 'os'
+import path from 'path'
```
Comment (ai) [inline create-reporter/index.js:3]: Can we move these import to `import { join, dirname } from 'path'` and `import { fileURLToPath } from 'url'` to keep consistency across the project?

Comment (Zn250g) [reply create-reporter/index.js:3]: Yes! Of course.

Sorry for not responding to the previous comments. I'm trying to make things work consistently and as expected. Lots of new things for me.

Thank you so much for your review.

**Comment by Zn250g** [comment]
Hi

It is quite possible that [the tests do not pass](https://github.com/logux/server/runs/5769073818?check_suite_focus=true#step:6:836) because of the deadlock [here](https://github.com/pinojs/thread-stream/blob/v0.15.1/index.js#L480).

Working on it.

**Pull request closed without merging**